packages/windows/vcruntime/vs2013/redist.txt

```diff
@@ -1,9 +1,14 @@
-This package contains Visual C++ Runtime files:
+Subject to Microsoft Licensing Product Use Rights [1], This package
+contains Visual C++ Runtime files as redistributable code:
+
+[1] http://download.microsoft.com/download/7/8/6/786cbfca-6826-4c58-9f4c-ee2067e619f5%2FPUR0604.pdf

```

Comment:
This PDF is Japanese.
It is OK if we don't prepare a link to the English version?

Reply:
Ah, it is better to prepare English version.

Reply:
I can't find english version PUR which is corresponding to Japanese PUR ...

Reply:
For VS 2013, we may use this document:
https://visualstudio.microsoft.com/license-terms/dn877550/